src/app/auth.service.ts

```diff
@@ -38,6 +39,13 @@ export class AuthService {
     return song;
   }
 
+  /** Returns an Observable for the songs liked by the signed-in user. */
+  getLikes(idToken): Observable<any> {
```

Comment:
Specify parameter types, so that the compiler checks that the function is being used correctly

```suggestion
  getLikes(idToken: string): Observable<Like[]> {
```